bpf/ut/icmp_too_big_test.go

```diff
@@ -43,7 +43,7 @@ func TestICMPTooBig(t *testing.T) {
 		pktR := gopacket.NewPacket(res.dataOut, layers.LayerTypeEthernet, gopacket.Default)
 		fmt.Printf("pktR = %+v\n", pktR)
 
-		checkICMPTooBig(pktR, ipv4, udp, natTunnelMTU-ethernetHeaderSize-20 /* compiled as WEP */)
+		checkICMPTooBig(pktR, ipv4, udp, natTunnelMTU-50 /* compiled as WEP */)
```

Comment:
Please can you add commenting here to explain what this test is testing, and in particular the magic numbers 64 and 50?

Also, do we have a loop somewhere that tests all of the possible UDP packet sizes?  Can we add that, if we don't have it?

Reply:
This only tests that `icmp_v4_too_big()` can convert a packet into a ICMP response.

There is another test that actually tests that we check if packet is oversized before we encap in `TestNATNodePort()` in `bpf/ut/nat_test.go`. Note that we are not able to test the (non-)gso scenarious at this level - all packets are non-gso.

Added comment a where the -50 comes from